charts/managed-identity-wallets/values.yaml

```diff
@@ -78,7 +98,7 @@ acapy:
   # The configuration of the AcaPy endorser instance
   endorser:
     # -- The url of the used Indy ledger
-    ledgerUrl: "https://idu.cloudcompass.ca"
+    ledgerUrl: "https://raw.githubusercontent.com/catenax-ng/product-core-schemas/main/genesis"
```

Comment:
Does this URL make sense for the default config? Would it also be a possibility to package that in some kind of nginx docker image, that you can then deploy with your app if i.e. 'localLedger=true' is set?

Reply:
Thank you. We are not using a local ledger, we just copied the genesis file from the scanner of the idunion test ledger, because it has a temporary certificate problem. I changed the values for now and in the future, this will be changed and the genesis files would be probably given directly to AcaPy instances instead of loading them from external urls.